### ISSUE TITLE
Codec support?

### ISSUE THREAD

**Issue by ceeeekay** [mschneider82/logstash-input-nsq#3]
Does this plugin support codecs? I can't read JSON messages from NSQ without the entire message being rewritten inside another message field. When testing with `stdout { codec => json {} }` on the publishing node, the message is formatted correctly.
```
output {
    nsq {
      nsqd => "127.0.0.1:4150"
      topic => "test"
      codec => json {}
    }
}
```
```
input {
  nsq {
    nsqlookupd => ["nsqlookupd1:4161","nsqlookupd2:4161","nsqlookupd3:4161"]
    topic => "test"
    channel => "to-filter"
    max_in_flight  => 128
    codec => json {}
  }
}
```
Output from consuming node:
```
{
    "@timestamp" => 2017-10-31T01:56:48.055Z,
      "@version" => "1",
       "message" => "{\"offset\":215954824,\"input_type\":\"log\",\"source\":\"/var/log/latency-test.log\",\"message\":\"1509415002.663\",\"type\":\"latency-test\",\"@timestamp\":\"2017-10-31T01:56:44.711Z\",\"@version\":\"1\" [...]
}
```

**Comment by mschneider82** [comment]
Not yet supported, in the meanwhile you can use:

```
filter {
 json {
    source => "message"
 }
 mutate {
   remove_field => "message"
 }
}
```

to decode the message

**Comment by mschneider82** [comment]
Fixed with 2.0.3, please update your plugin
New default: json

**Comment by ceeeekay** [comment]
Works great. Thanks very much!
